final_project/snapvisite/tests/factories.py

```diff
@@ -0,0 +1,75 @@
+import factory
+from faker import Faker
+from django.contrib.auth import get_user_model
+from snapvisite.models import Company, Category
+profile = get_user_model()
+fake = Faker()
+
+
+"""User Factories"""
+
+
+class UserFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = profile
+
+    email = 'user@test.com'
+    user_name = 'nickname'
+    first_name = fake.first_name()
+    last_name = fake.last_name()
+    is_active = True
+    is_staff = False
+    is_superuser = False
+    phone_number = '+48517856890'
+    confirm = True
+    print("user is created")
+
+
+class SuperUserFactory(factory.django.DjangoModelFactory):
```

Comment:
Ja bym to złączył w jedną factorkę, gdzie możesz wybrać czy chcesz żeby to był superuser, czy nie